src/Model/ISO2ROI.py

```diff
@@ -0,0 +1,292 @@
+                ds.save_as(file_path)
+
+                # Add RT Struct file path to patient dict container
+                patient_dict_container.filepaths['rtss'] = file_path
+                filepaths = patient_dict_container.filepaths
+
+                # Add RT Struct dataset to patient dict container
+                patient_dict_container.dataset['rtss'] = ds
+                dataset = patient_dict_container.dataset
+
+                # Set some patient dict container attributes
+                patient_dict_container.set("file_rtss", filepaths['rtss'])
+                patient_dict_container.set("dataset_rtss", dataset['rtss'])
+
+                dicom_tree_rtss = DicomTree(filepaths['rtss'])
+                patient_dict_container.set("dict_dicom_tree_rtss",
+                                           dicom_tree_rtss.dict)
+
+                dict_pixluts = ImageLoading.get_pixluts(
+                    patient_dict_container.dataset)
+                patient_dict_container.set("pixluts", dict_pixluts)
+
+                rois = ImageLoading.get_roi_info(ds)
+                patient_dict_container.set("rois", rois)
+
+                patient_dict_container.set("selected_rois", [])
+                patient_dict_container.set("dict_polygons_axial", {})
+
+        # Get isodose levels to turn into ROIs
+        self.get_iso_levels()
+
+        # Stop loading
+        if interrupt_flag.is_set():
+            print("Stopped ISO2ROI")
+            return False
+
+        # Calculate dose boundaries
+        progress_callback.emit(("Calculating Boundaries", 50))
+        boundaries = self.calculate_boundaries()
+
+        # Return if boundaries could not be calculated
+        if not boundaries:
+            print("Boundaries could not be calculated.")
+            return
+
+        # Stop loading
+        if interrupt_flag.is_set():
+            print("Stopped ISO2ROI")
+            return False
+
+        progress_callback.emit(("Generating ROIs", 75))
+        self.generate_roi(boundaries, progress_callback)
+        progress_callback.emit(("Reloading Window. Please Wait...", 95))
+
+    def get_iso_levels(self):
+        """
+        Opens /data/csv/isodoseRoi.csv to find the isodose levels that
+        the user wants turned into ROIs. Creates a dictionary where the
+        key is the isoname, and the value is a list containing a boolean
+        (cGy: 0, %: 1) and an integer (cGy/% value).
+        """
+
+        # Clear self.isodose_levels (in case ISO2ROI has already been
+        # run this session)
+        self.isodose_levels = {}
+
+        # Open isodoseRoi.csv
+        with open('data/csv/isodoseRoi.csv', "r") as fileInput:
+            for row in fileInput:
+                items = row.split(',')
+                self.isodose_levels[items[2]] = [items[1] == 'cGy',
+                                                 int(items[0])]
+
+    def calculate_boundaries(self):
```

Comment:
maybe rename this to `calculate_roi_boundaries`?

Reply:
This function does not calculate ROI boundaries, it calculates isodose boundaries. However, it will be renamed to `calculate_isodose_boundaries`. As for the print statements, since enough people have commented on them they will be removed.

Reply:
Changes have been made and added to the PR.